low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -0,0 +1,360 @@
+        if self.args.bmc_user:
+            self.BMC_USER = self.args.bmc_user
+        if self.args.bmc_passwd:
+            self.BMC_PASSWD = self.args.bmc_passwd
+        if self.args.storage_type:
+            self.STORAGE_TYPE = self.args.storage_type
+        if self.args.server_type:
+            self.SERVER_TYPE = self.args.server_type
+        if self.args.cluster_nodes:
+            self.CLUSTER_NODES = self.args.cluster_nodes
+        if not (self.TARGET_BUILD or self.RPMS_PATH):
+            argParser.print_help()
+            raise DeployError(1, "Target build url or local RPMS path is missing")
+        if self.RPMS_PATH and not os.path.exists(self.RPMS_PATH):
+            raise DeployError(1, "RPMS path is not found.")
+
+    def cleanup(self):
+        """Stop sspl service and remove SSPL RPMS."""
+        cmd = "systemctl stop sspl-ll.service"
+        _ = os.system(cmd)
+        cmd = "yum remove -y cortx-sspl.noarch"
+        returncode = os.system(cmd)
+        if returncode != 0:
+            raise DeployError(returncode, "Unable to remove sspl RPMS. %s",
+                              "Cleanup failed.")
+        sys.exit(0)
+
+    def setup_yum_repos(self):
+        """Setup common, 3rd_party and build specific repos."""
+        if not self.setup_repo:
+            return
+        setup_repo_url = self.CORTX_MONITOR_BASE_URL + \
+            "/low-level/files/opt/seagate/sspl/setup/setup_yum_repos.py"
+        setup_repo_txt = requests.get(setup_repo_url, allow_redirects=True)
+        with open('setup_yum_repos.py', 'w+') as fle:
+            fle.write(setup_repo_txt.content)
+        os.chmod("setup_yum_repos.py", 0o775)
+
+        if self.TARGET_BUILD:
+            cmd = "./setup_yum_repos.py -t %s" % (self.TARGET_BUILD)
+            returncode= os.system(cmd)
+        else:
+            cmd = "./setup_yum_repos.py"
+            returncode = os.system(cmd)
+        if returncode != 0:
+            raise DeployError(returncode, "%s - Failed to setup yum repos.",
+                "setup_yum_repos.py")
+
+    def install_dependencies(self):
+        """Install common and 3rd_party application."""
+        if self.install_3rd_party_packages:
+            print("INFO: INSTALLING python3...")
+            cmd = "yum install -y python3"
+            _ = os.system(cmd)
+
+            # cortx-py-utils dependency
+            cmd = "python3 -m pip install toml pyyaml"
+            _ = os.system(cmd)
+
+            print("INFO: INSTALLING cortx-py-utils...")
+            pkg_name="cortx-py-utils"
+            # TODO: EOS-17072 - last_successful location is not having recent
+            # cortx-py-utils package. Untill then proceeding with static one.
+            # build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/cortx-utils_last_successful/"
+            build_url = "http://cortx-storage.colo.seagate.com/releases/eos/github/main/centos-7.8.2003/cortx-utils_99/prod/cortx_iso/"
```

Comment:
you can use **"http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"** . its having recent cortx-utils pkg.

and also before installing cortx-utils , we need to install **"yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel"** these pkgs. This is required for conf stor cli.

Reply:
As commented in the script,
`# TODO: EOS-17072 - last_successful location is not having recent
# cortx-py-utils package. Untill then proceeding with static one.`
This ticket is just now resolved. we will update it.

Also the mentioned support packages are only required to build cortx-py-utils rom.
As we directly installs cortx-py-utils rpm from URL, we don't need to install these support packages I guess.

Reply:
"yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel "
 if we install cortx-utils without this pkgs , then some pip dependencies will not get installed, and conf stor cli will not work. these pkgs are dependacies of pip pkgs.

Reply:
`TODO: EOS-17072 - last_successful location is not having recent cortx-py-utils package. Untill then proceeding with static one`
This is addressed by RE. Now last_successful is having recent cortx-py-utils RPMS.
@satish-seagate , please change this build URL and remove TODO.